FurCoNZ/Services/OrderService.cs

```diff
@@ -45,7 +45,79 @@ public Task<DateTimeOffset> ReserveTicketsForPurchaseAsync(IDictionary<int, int>
         public async Task<IEnumerable<Order>> GetUserOrders(User user, CancellationToken cancellationToken = default)
         {
             // TODO: Should we support pagination?
-            return await _db.Orders.Where(o => o.OrderedBy == user).ToListAsync(cancellationToken);
+            return await _db.Orders
+                .Include(o => o.TicketsPurchased)
+                .ThenInclude(t => t.TicketType)
+                .Where(o => o.OrderedBy == user).ToListAsync(cancellationToken);
```

Comment:
Can put `.Where()` and `.ToListAsync()` on separate lines so the latter is more obvious